### PR TITLE
add doc for contributor data sources

### PR DESCRIPTION
**Pull request by nberard**

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

File: documentation/openAPI.yaml [hunk 1]
```diff
@@ -420,6 +420,46 @@ paths:
           description: Error
           schema :
             $ref: '#/definitions/error'
+  /contributors/{contributor_id}/data_sources:
```
Comment (pbougue) [inline documentation/openAPI.yaml]: missing a blank line

File: documentation/openAPI.yaml [hunk 2]
```diff
@@ -420,6 +420,46 @@ paths:
...
+        200:
+          description: Requested contributor data source
+    post:
+      summary: Create a new data source for a contributor
+      tags:
+        - contributors
+      parameters:
+        - in: body
+          name: body
+          description: Data source object that needs to be added to Tartare
```
Comment (pbougue) [inline documentation/openAPI.yaml]: maybe "needs to be added to `given contributor`"

File: documentation/openAPI.yaml [hunk 3]
```diff
@@ -495,6 +535,27 @@ definitions:
...
+        type: array
+        items:
+          $ref: "#/definitions/data_source_type"
+
+  data_source_type:
+    type: object
+    properties:
+      id:
+        type: string
+        example: 5810cb2d3c96b7000108d074
```
Comment (pbougue) [inline documentation/openAPI.yaml]: `tram_lyon` would be more readable

File: documentation/openAPI.yaml [hunk 4]
```diff
@@ -495,6 +535,27 @@ definitions:
...
+      name:
+        type: string
+        example: my data source
```
Comment (pbougue) [inline documentation/openAPI.yaml]: `Tram feed for Lyon` would be more readable

**Pull request merged**